lib/add-allegation-data.js

```diff
@@ -4,11 +4,22 @@ const _ = require('lodash');
 const fieldsMap = require('./ims-hof-fields-map.json');
 const valuesMap = require('./ims-hof-values-map.json');
 
+const addAdditionalPeople = (eform, additonalpeople) => {
+  if (additonalpeople.length > 0) {
+    additonalpeople[0].map(item => {
+      eform.EformFields.push({FieldName: item.FieldName, FieldValue: item.FieldValue});
+    });
+    if (additonalpeople.length > 1) {
+      eform.AdditionalPeople = additonalpeople.splice(1);
+    }
+  }
+};
+
 const addFieldToEform = (eform, fieldName, fieldValue) => {
   if (fieldName === 'images') {
     eform.Attachements = fieldValue;
```

Comment:
ims-resolver looks for `msg.Attachments` will need to remove the extra 'e' in 'Attachements' for the URLs to be added properly.